samples/Nancy.Demo.Async/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
+  "authors": [ "Andreas Håkansson, Steven Robbins and contributors" ],
+  "tags": [ "Nancy" ],
+  "projectUrl": "http://nancyfx.org",
+  "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
+  "iconUrl": "http://nancyfx.org/nancy-nuget.png",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Nancy": { "target": "project" },
+    "System.Net.Http": "4.0.1-beta-23516"

```

Comment:
This is a framework assembly in .NET 4.5, right? Should probably be moved to the `frameworks` section.